src/components/Dashboard/FilingHistoryList/HeaderActions.vue

```diff
@@ -131,9 +132,9 @@ export default class HeaderActions extends Mixins(AllowableActionsMixin) {
 
       case FilingTypes.CHANGE_OF_ADDRESS:
       case FilingTypes.CHANGE_OF_DIRECTORS:
-        if (this.isBenBcCccUlc) {
-          // correction via Edit UI if current type is BC, CC, ULC, or BEN
-          // To-Do for the future: Revisit this when we do Coop corrections in Edit UI
+      case FilingTypes.SPECIAL_RESOLUTION:
+        if (this.isBenBcCccUlc || (filing.name === FilingTypes.SPECIAL_RESOLUTION && this.isCoop)) {
+          // correction via Edit UI if current type is BC, CC, ULC, BEN or coop special resolution correction
```

Comment:
I get what you're trying to do here (SR corrections go to Edit UI but Coop COA and COD corrections use the else case below and are local corrections) and I think that's correct.

But this is overly complicated and confusing. How about just moving the Special Resolution case to line 128?

Reply:
Jia, please let me resolve the conversations that I started -- an open conversation shows me the thing I have to re-review (otherwise I have to re-review everything to make sure any issues were fixed). Thanks!